packages/block-library/src/search/edit.js

```diff
@@ -505,6 +420,80 @@ export default function SearchEdit( {
 							</ToggleGroupControl>
 						</VStack>
 					</ToolsPanelItem>
+					<ToolsPanelItem
+						hasValue={ () => ! showLabel }
+						label={ __( 'Show search label' ) }
+						onDeselect={ () => {
+							setAttributes( {
+								showLabel: true,
+							} );
+						} }
+						isShownByDefault
+					>
+						<ToggleControl
+							__nextHasNoMarginBottom
+							checked={ showLabel }
+							label={ __( 'Show search label' ) }
+							onChange={ ( value ) =>
+								setAttributes( {
+									showLabel: value,
+								} )
+							}
+						/>
+					</ToolsPanelItem>
+					<ToolsPanelItem
+						hasValue={ () => !! buttonUseIcon }
+						label={ __( 'Use button with icon' ) }
+						onDeselect={ () => {
+							setAttributes( {
+								buttonUseIcon: false,
+							} );
+						} }
+						isShownByDefault
+					>
+						<ToggleControl
+							__nextHasNoMarginBottom
+							checked={ buttonUseIcon }
+							label={ __( 'Use button with icon' ) }
+							onChange={ ( value ) =>
+								setAttributes( {
+									buttonUseIcon: value,
+								} )
+							}
+						/>
+					</ToolsPanelItem>
+					<ToolsPanelItem
+						hasValue={ () => buttonPosition !== 'button-outside' }
+						label={ __( 'Change button position' ) }
```

Comment:
```suggestion
						label={ __( 'Button position' ) }
```

It's a good idea to avoid including verbs in select list labels.